### PR TITLE
UIORGS-84 Fix capitalization of phrases

### PR DESCRIPTION
**Pull request by AndreiShumski2**
## Purpose
Fix capitalization of phrases in ui-organizations
https://issues.folio.org/browse/UIORGS-84
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: translations/ui-organizations/en.json [hunk 1]
```diff
@@ -10,16 +10,16 @@
   "data.contactTypes.address": "Address",
   "data.contactTypes.addressLine1": "Address 1",
   "data.contactTypes.addressLine2": "Address 2",
-  "data.contactTypes.stateProviceOrRegion": "State/Province/Region",
-  "data.contactTypes.zipOrPostalCode": "Zip/Postal code",
+  "data.contactTypes.stateProviceOrRegion": "State/province/region",
```
Comment (aliaksei-chumakou) [inline translations/ui-organizations/en.json]: Are you sure about these?

File: translations/ui-organizations/en.json [hunk 2]
```diff
@@ -10,16 +10,16 @@
...
+  "data.contactTypes.zipOrPostalCode": "Zip/postal code",
```
Comment (aliaksei-chumakou) [inline translations/ui-organizations/en.json]: the same, usually `Postal code` goes with capital

File: translations/ui-organizations/en.json [hunk 3]
```diff
@@ -400,7 +400,7 @@
   "contactPeople.addressLine1": "Address 1",
   "contactPeople.addressLine2": "Address 2",
   "contactPeople.city": "City",
-  "contactPeople.stateRegion": "State/Prov./Region",
+  "contactPeople.stateRegion": "State/prov./region",
```
Comment (aliaksei-chumakou) [inline translations/ui-organizations/en.json]: the same

File: translations/ui-organizations/en.json [hunk 4]
```diff
@@ -409,7 +409,7 @@
   "contactPeople.addresses.addressLine1": "Address 1",
   "contactPeople.addresses.addressLine2": "Address 2",
   "contactPeople.addresses.city": "City",
-  "contactPeople.addresses.stateRegion": "State/Prov./Region",
+  "contactPeople.addresses.stateRegion": "State/prov./region",
```
Comment (aliaksei-chumakou) [inline translations/ui-organizations/en.json]: the same

File: translations/ui-organizations/en.json (hunk 1 above)
Comment (AndreiShumski2) [reply translations/ui-organizations/en.json]: Actually no. Reverted these changes

File: translations/ui-organizations/en.json (hunk 2 above)
Comment (AndreiShumski2) [reply translations/ui-organizations/en.json]: Ok

**Pull request merged**